app/views/images/_complete.html.erb

```diff
@@ -8,8 +8,10 @@
 
     <%= help_text t("images.help.#{image.model_name.singular}") %>
 
-    <%= link_to delete_path, class: "prx-btn-help" do %>
-      <span class="material-icons">delete</span>
+    <% if policy(@podcast).update? %>
```

Comment:
I think there are Image policies, for every flavor - so can probably just `policy(image).destroy?`.

And the error/processing partials also need this.